src/components/Form.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../style/Form.css';
+
+function form() {
```

Comment:
Please rename your "form component" as "CreateNewBook" to make it more meaningful and descriptive 👍🏻